python/ml4ir/base/data/relevance_dataset.py

```diff
@@ -28,6 +29,9 @@ def __init__(
         use_part_files: bool = False,
         parse_tfrecord: bool = True,
         logger: Optional[Logger] = None,
+        keep_additional_info: int = 0,
+        gl_2_clicks: int = 1,
```

Comment:
@mohazahran Let's see if we can add `gl_2_clicks` to the feature_config itself under label.